### PR TITLE
Add more copyright headers for files we added or modified

### PR DESCRIPTION
**Pull request by mgianatagh**
Add more copyright headers for files we added or modified

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -7,7 +10,3 @@ options.forkOptions.memoryMaximumSize=2g
 # See https://docs.gradle.org/current/userguide/upgrading_version_6.html#duplicate_project_names_may_cause_publication_to_fail
 systemProp.org.gradle.dependency.duplicate.project.detection=false
 
-#systemProp.http.proxyHost=www-proxy-hqdc.us.oracle.com
```
Comment (kminder) [inline gradle.properties:10]: Doesn't seem like this change should be included with the other chagnes.

Comment (mgianatagh) [reply gradle.properties:10]: Mark wants us to also remove commented out content.  This line was commented out, and contains internal information that must be removed.

The real purpose of this PR is to get the cert-manager repo ready for Mark to do a full review of our changes.

**Comment by mgianatagh** [comment]
Roberto pointed out that I used the wrong copyright text.  I copied from the crd-generator repo and it has not been updated with the latest copyrights yet.  I am fixing and will repost.

**Comment by mgianatagh** [comment]
I pushed the fixes to the copyright text

File: gradle.properties (hunk 1 above)
Comment (markxnelson) [reply gradle.properties:10]: i'm ok to include this here on this occassion, though i support the general sentiment

**Pull request merged**